inductiva/tasks/streams.py

```diff
@@ -51,20 +54,26 @@ def __init__(self,
     def __setup_websocket(self):
         """Initialize the websocket app, with the callbacks within the class."""
         return websocket.WebSocketApp(self.websocket_url,
+                                      header={"X-API-Key": inductiva.api_key},
                                       on_open=self.__on_open,
                                       on_error=self.__on_error,
                                       on_close=self.__on_close,
                                       on_message=self.__on_message)
 
     def __on_message(self, unused_ws, message):
-        print(message, file=self.fout)
+        data = json.loads(message)
```

Comment:
it would be useful to have a schema of the message's json.

Reply:
Indeed! Maybe in the next steps we can provide that.